src/views/GovernanceReviewTeam/BusinessCaseReview/index.test.tsx

```diff
@@ -26,6 +26,7 @@ describe('The GRT business case review', () => {
     },
     businessNeed: 'Mock business need',
     cmsBenefit: 'Mock CMS benefit',
+    currentSolutionSummary: 'Mock current solution summary',
     priorityAlignment: 'Mock priority alignment',
     successIndicators: 'Mock success indicators',
     asIsSolution: {
```

Comment:
Final change (and this one's minor), but it looks like this BusinessCaseReview test component still has some references to `asIsSolution` - guessing we can just get rid of these?

Reply:
I'll look into it but I assume we can

Reply:
I replaced it with a preferred solution object, had to update the snapshot as well.